editor-blender/panels/lightdance/__init__.py

```diff
@@ -1,10 +1,17 @@
+from typing import Any
+
 import bpy
+from bpy.types import Context, UILayout
```

Comment:
I don't think this is necessary, using `bpy` types without `bpy.*` could be confusing IMO.